<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Nisheshs-MacBook-Air.local" failures="1" tests="2" name="Test(failed)" time="28.081" errors="0" timestamp="2025-04-18T13:52:30 CDT">
  <testcase classname="codenbox.testCases.LoginTest" name="verify_login" time="10.53">
    <skipped/>
  </testcase> <!-- verify_login -->
  <testcase classname="codenbox.testCases.LoginTest" name="verify_login" time="10.618">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at codenbox.testCases.LoginTest.verify_login(LoginTest.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
</testsuite> <!-- Test(failed) -->
